includes/bootstrap.inc

```diff
@@ -273,7 +274,7 @@ function drush_bootstrap_validate($phase) {
         break;
       }
       if ($phase_index > $validated_phase) {
-        $current_phase .= '_validate';
+        $current_phase .= 'Validate';
```

Comment:
Also, the only bootstrap class in use in Drush 9 is DrupalBoot8, so I think the impact here is negligible.